test/test.js

```diff
@@ -10,8 +13,31 @@ jasmine.loadConfig({
     ]
 });
 
-var runner =  new JsReporters.JasmineAdapter(jasmine);
-var testReporter = new JsReporters.TestReporter(runner, referenceData.Jasmine);
+var jasmineRunner = new JsReporters.JasmineAdapter(jasmine.env);
+jasmine.addReporter({}); // Suppress the default reporter
+var jasmineTestReporter = new JsReporters.TestReporter(jasmineRunner, referenceData.Jasmine);
 
 jasmine.execute();
-process.exit(testReporter.ok ? 0 : 1);
+if (!jasmineTestReporter.ok) {
+    throw new Error("Tests for JasmineAdapter failed!");
+}
+
+//TODO: Use a proper test framework?
```

Comment:
Using a test framework with proper async support would make this test easier to understand. Choosing between QUnit, Jasmine, Mocha, Buster or Intern is very meta here. Worth the effort?